test/e2e/README.md

```diff
@@ -192,6 +192,49 @@ You can load the new image into the kind cluster using the command below:
 kind load docker-image projects.registry.vmware.com/antrea/antrea-ubuntu:latest --name <kind_cluster_name>
 ```
 
+## Running the e2e tests on existing K8s cluster
+
+If you already have a K8s cluster on which Antrea has been pushed, the following steps should be done for running the e2e tests.
+
+First, you should create the ssh config file and add the ssh information of each Node in the cluster to it. You can add the ssh information like the example below:
+
+```bash
+Host Control-Plane-Node
+    HostName <Control-Plane-IP>
+    Port 22
+    user ubuntu
+    IdentityFile /home/ubuntu/.ssh/id_rsa
+Host Worker-Node
+    HostName <Worker-Node-IP>
+    Port 22
+    user ubuntu
+    IdentityFile /home/ubuntu/.ssh/id_rsa
+```
+
+Make sure `Host` of each Node must be the same as the corresponding Node name. The `Port` is the default port of ssh service. As for the `IdentityFile`, add the location of the private key (`id_rsa` in example) to it.
+
+Beside the configuration of the ssh connection, make sure you have added the public key to `authorized_keys` of each Node and set `PubkeyAuthentication` and `RSAAuthentication` of ssh service to `yes`.
+
+Second, when running the e2e tests, the configuration information of cluster will be used, so make sure you have copied the config file of the cluster to your localhost.
```

Comment:
```suggestion
Second, when running the e2e tests, kubeconfig of the cluster will be used, so make sure you have copied it to the right location, e.g. `$HOME/.kube/config` or will use it directly from `/etc/kubernetes/admin.conf`.
```

I guess you are talking about kubeconfig. Please correct me if I'm wrong.

Reply:
I don't find the subject of the statement "or will use it directly from `/etc/kubernetes/admin.conf`", and it doesn't seem necessary. Maybe "or the path specified by `-remote.kubeconfig`".